content/en/users/compute/cloud-compute/auth/_index.md

```diff
@@ -151,5 +151,55 @@ export OS_TOKEN=$(fedcloud openstack --site <NAME_OF_SITE> --vo <NAME_OF_VO> \
 
 ### Using ssh-oidc
 
-[ssh-oidc](https://github.com/EOSC-synergy/ssh-oidc) is a set of tools that
-allows ssh with OIDC.
+[ssh-oidc](https://github.com/EOSC-synergy/ssh-oidc) is a collection of
+tools that enable ssh using OpenID-Connect access tokens.
+
+Most of the individual tools work standalone, but they may be combined to
+establish complex installations with dynamic account provisioning in
+multi-OP and multi-VO contexts.
+
+The tools share these common design criteria:
+
+- None of the ssh-client or ssh-server components will be modified
+- Do not store state whenever possible
+  - Single exception: federated-user -> local-user mapping is stored in
+    `passwd`
+- Small components that work individually (one tool for one job)
+
+#### Installation (client-side)
+
+`ssh-oidc` does not require specific clients per se. OIDC Access-Tokens
+may simply be passed, when ssh prompts for the "Access Token".
+Unfortunately, the most-popular ssh-client does not support access-tokens
+which are longer than 1024 bytes _and_ EGI uses tokens that are typically
+longer than 1024 bytes.
+
+Hence, we suggest using [mccli](https://mccli.readthedocs.org) (`pip
+install mccli`) on Unix/Mac. Windows users should use the putty
+`oidc-plugin`, which conveniently ships with
+[oidc-agent-for-windows](http://repo.data.kit.edu/windows/oidc-agent/).
+
+For conveniently obtaining access tokens,
+[oidc-agent](https://indigo-dc.gitbook.io/oidc-agent) is probably the most
+convenient choice.
```

Comment:
conveniently and convenient in the same sentence?

Reply:
Well, oidc-agent is super convenient :)
I'll change that with the above comment.